Yoast/yoast-components
# Sidebar styling tweaks

## Summary

This PR can be summarized in the following changelog entry:

* Changed the collapsible border color to the one gutenberg uses.
* Changed the titles of the analytics to be not bold but normal and to set the text color to black/ very dark grey .
* Removed the max-width from the analysis component to remove the wrapping of the analysis results on big screens
* Added a dropshadow to the input fields in the sidebar.

## Relevant technical choices:

* Adding a color in the `colors.json`.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and link it into [wordpress seo](https://github.com/Yoast/wordpress-seo).
* Go to a post and edit it.
* Verify the sidebar looks like the image below and the changelog entries are correct.
<img width="259" alt="cat___ _qwerty_ _wordpress" src="https://user-images.githubusercontent.com/5509155/44033455-baddf470-9f0a-11e8-9d20-02e8c5c0b9ed.png">



Fixes https://github.com/Yoast/wordpress-seo/issues/10596
